index.html

```diff
@@ -859,79 +911,88 @@ <h2 class="secondary-heading">FAQS</h2>
 
     <!-- Footer -->
     <footer>
-      <section class="call-action container">
+      <section class="cta-homepage container">
         <h2 class="secondary-heading">JOIN OUR COMMUNITY</h2>
-        <p>
+        <p class="cta-homepage-text">
           Lorem Ipsum is simply dummy text of the printing and typesetting
           industry.
         </p>
 
-        <div class="footer-btn-box">
-          <a href="#" class="basic-btn media-btn">
-            <img src="./img/btn-icons/discord-btn.svg" alt="" />
+        <div class="cta-homepage-btns">
+          <a href="#" class="btn-basic btn-media">
+            <svg class="icon discord-btn" width="24" height="24">
+              <use xlink:href="sprite.svg#discord-btn"></use>
+            </svg>
             DISCORD
           </a>
-          <button class="basic-btn btn-light">WHITELIST NOW</button>
+          <button class="btn-basic btn-light" type="button">
+            WHITELIST NOW
+          </button>
         </div>
 
+        <form action="" method="post" class="home-newsletter">
+          <label for="email" class="visually-hidden">Email:</label>
+          <input
+            class="newsletter-field field-validation"
+            type="email"
+            name="email"
+            id="email"
+            placeholder="Enter your email"
+            autocomplete="email"
+            required
+            pattern="[a-z0-9._%+\-]+@[a-z0-9.\-]+\.[a-z]{2,}$"
+          />
+          <input class="btn-light btn-basic" type="submit" value="SUBSCRIBE" />

```

Comment:
Why didn't you use tag `button` here? Your classes say it should be a button )

Reply:
What is the difference and what is better to use in practice? 

I thought `<input type="submit">` and `<button type="submit">` are the same or interchangeable. 

thanks in advance.

Reply:
Thank you for explanation. 

#fixed